scripts/generator/graphql_generator/csharp/Editor/ShopifyOnboardingPanel.cs.erb

```diff
@@ -73,9 +73,10 @@ namespace <%= namespace %>.SDK {
                 Styles.BodyStyle
             );
 
-            if (GUILayout.Button("Start your free trial", Styles.ButtonStyle))
-            Application.OpenURL("https://www.shopify.ca/signup");
-
+            if (GUILayout.Button("Start your free trial", Styles.ButtonStyle)) {
+                Application.OpenURL("https://www.shopify.ca/signup?signup_types=unity_buy_sdk,<%= publish_destination %>");
```

Comment:
`.ca` or `.com`? They probably both work, but maybe signup country domains are used somewhere in data land, so `.com` might be safer (since it redirects to country codes when applicable). That said, maybe brochure's nginx has redirects from `.ca` too.